apisix/plugins/authz-keycloak.lua

```diff
@@ -61,8 +61,13 @@ local schema = {
         cache_ttl_seconds = {type = "integer", minimum = 1, default = 24 * 60 * 60},
         keepalive = {type = "boolean", default = true},
         keepalive_timeout = {type = "integer", minimum = 1000, default = 60000},
-        keepalive_pool = {type = "integer", minimum = 1, default = 5}
-    },
+        keepalive_pool = {type = "integer", minimum = 1, default = 5},
+        access_token_expires_in = {type = "integer", minimum = 1, default = 300},
```

Comment:
We can remove the `or` logic now?
https://github.com/apache/apisix/blob/ecf08c6dd50c3ea3e7bf3c4723ada5e90bbeae13/apisix/plugins/authz-keycloak.lua#L318